test/setVersionByQueryParam.test.js

```diff
@@ -66,15 +63,14 @@ test('we can set a version on the request object by request query parameters', t
   })
 })
 
-
 test('we can manually set a specific version to be string', t => {
-  const versionNumber = '1'
+  const versionNumber = '1.0.0'
 
   t.context.req.query['api-version'] = versionNumber
   const middleware = versionRequest.setVersionByQueryParam()
 
   middleware(t.context.req, {}, () => {
-    t.is(t.context.req.version, versionNumber)
+    t.is(t.context.req.version, versionRequest.formatVersion(versionNumber))
```

Comment:
why do you think we need to use formatVersion here?
I think the assertion can (and probably should) stay as it was originally:
```js
t.is(t.context.req.version, versionNumber)
```

the point being that we don't want to count on internal logic of the library in our assertions.

Reply:
If you agree, then we need to change it in the other tests as well as it is repeating there too (for example: https://github.com/gagyibenedek/express-version-request/blob/997b96786da3b3a4d99f5abbf4f1c8f3c6d0e46f/test/setVersionByAcceptHeader.test.js#L52)

Reply:
Agreed to work this out in a later PR through test spies like those provided by the sinon library.